src/misc/unicode.cpp

```diff
@@ -985,9 +1252,9 @@ static void import_config_main(const std_fs::path& path_root)
 
 	uint16_t curent_code_page = 0;
 
-	config_mappings_t new_config_mappings;
-	config_duplicates_t new_config_duplicates;
-	config_aliases_t new_config_aliases;
+	config_mappings_t   new_config_mappings   = {};
+	config_duplicates_t new_config_duplicates = {};
+	config_aliases_t    new_config_aliases    = {};
```

Comment:
when all the types are the same, the variable names will coincidentally be left-aligned, so visually they appear all as the same type.

In this case my eyes quickly see the same nice variable alignment:
```c++
..   new_config_a = {};
..   new_config_b = {};
..   new_config_c = {};
```

However, the overriden variable-name alignment has fooled me, because they are infact all different types.

Please let the formatter run here - yes, it'll break variable alignment because the types are different (with different lengths), but that's making it easier to differentiate them.

Reply:
OK, reverted the change.